src/table/paginator.scss

```diff
@@ -3,12 +3,13 @@
 @import '../style/index';
 @import '../icon/index';
 @import '../button/index';
+@import '../text/index';
 @import './variables';
 
 @mixin iui-paginator {
   @include iui-reset;
   display: flex;
-  justify-content: space-around;
+  justify-content: space-between;
```

Comment:
Why was this changed?

Reply:
I don't recall, it may have had something to do with the responsive design I was trying out.  But looking at [this image from CSS-Tricks' article](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#justify-content), I believe `space-between` is what we actually want.
![image](https://user-images.githubusercontent.com/849817/144279611-8c9923cb-c1f0-4d92-bcb4-99609bc0c4db.png)